playbook/appherd/roles/mod_wsgi/tasks/main.yml

```diff
@@ -1,68 +1,25 @@
 ---
-# File: roles/python3/tasks/main.yml
-# Created: 5/10/17
-# Author: '@ekivemark'
-# download and install python3
-
-- name: "create download directory"
-  file:
-    dest: "/home/ec2-user/download"
-    state: directory
-    mode: 0600
-    owner: "ec2-user"
-    group: "ec2-user"
-  register: app_download_dir
-
-- name: "Download mod_wsgi_{{ mod_wsgi_ver_full }}"
-  become_user: "{{ remote_admin_account }}"
-  become: yes
-  get_url:
-    url: "https://codeload.github.com/GrahamDumpleton/mod_wsgi/tar.gz/{{ mod_wsgi_ver_full }}"
-    dest: "/home/ec2-user/download/mod_wsgi_{{ mod_wsgi_ver_full }}.tar.gz"
-    remote_src: True
-
-- name: "Unarchive mod_wsgi_{{ mod_wsgi_ver_full }}"
-  become_user: "{{ remote_admin_account }}"
-  become: yes
-  shell: "cd /home/ec2-user/download ; /usr/bin/tar -xzvf /home/ec2-user/download/mod_wsgi_{{ mod_wsgi_ver_full }}.tar.gz ; cd /home/ec2-user/download/mod_wsgi-{{ mod_wsgi_ver_full }} "
-  register: mod_wsgi_info
-
-- name: "Running configure for mod_wsgi"
-  become_user: "{{ remote_admin_account }}"
+- name: "install mod_wsgi"
   become: yes
-  shell: "cd /home/ec2-user/download/mod_wsgi-{{ mod_wsgi_ver_full }} ; make clean ; make uninstall ; make -n install ; ./configure --with-python={{ python_bin_dir }}/python{{ python_ver }} --with-apxs=/usr/bin/apxs "
-
-- name: "Prepare to refresh ld cache"
   become_user: "{{ remote_admin_account }}"
-  become: yes
-  lineinfile:
-    dest: "/etc/ld.so.conf"
-    line: "/usr/local/lib"
-    state: present
+  shell: |
+   source {{ cf_app_py_virtual_env }}/bin/activate && pip install mod_wsgi
```

Comment:
should we lock mod_wsgi to a version?

Reply:
I also think that is possibly a good idea.

Another alternative - could we just add this to our requirements file? Any reason to install this one module differently?

Reply:
I'll update to lock to the latest version (4.6.4 at time of writing).

It probably makes the most sense to do this here, since Apache needs to be installed first and we need to set the `APXS` env var before the install.

It could be moved to requirements, but that would necessitate further refactoring of the ansible roles/the build process in general.